packages/paste-core/components/timeline/src/TimelineItemIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from "@twilio-paste/box";
+import { IconWrapper } from "@twilio-paste/icons/esm/helpers/IconWrapper";
+import * as React from "react";
+
+import { TimelineContext } from "./TimelineContext";
+import type { TimelineItemIconProps } from "./types";
+
+const TimelineItemIcon = React.forwardRef<HTMLElement, TimelineItemIconProps>(
+  ({ icon: Icon, as, display, element = "TIMELINE_ITEM_ICON", color }, ref) => {
+    const { orientation } = React.useContext(TimelineContext);
+
+    return (
+      <Box element={`${element}_WRAPPER`}>
+        {Icon ? (
+          <Box marginY="space20">
+            {/* @ts-expect-error some error in size*/}
```

Comment:
has this typing been fixed? Any reason we have to ignore this?

Reply:
+1, what's the error here